OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -263,7 +263,7 @@ public void onEventMainThread(DiscussionThreadUpdatedEvent event) {
     @SuppressWarnings("unused")
     public void onEventMainThread(DiscussionCommentPostedEvent event) {
         // If a new response/comment was posted in a listed thread, we need to update the list
-        for (int i = 0; i < discussionPostsAdapter.getCount(); ++i) {
+        for (int i = 0; i < discussionPostsAdapter.getItemCount(); ++i) {
```

Comment:
I would suggest updating the `++i` to `i++` everywhere because now we are utilizing the list position and pre incremental value is causing the app to crash.

Reply:
Still some are pending.